crates/runtime/src/env/default/address_compute.rs

```diff
@@ -1,7 +1,7 @@
 use crate::env::{self, traits};
 
 use env::{ExtSpawnApp, Template};
-use traits::{AppAddressCompute, TemplateAddressCompute};
+use traits::AddressLocator;
```

Comment:
I'm not sure the `locator` is the right term.
It's really a computation process here